pages/emotion/[id].js

```diff
@@ -14,6 +14,16 @@ export default function EmotionDetailPage({ entries }) {
   if (!emotionEntry) {
     return <p>No emotion entry found.</p>;
   }
+  function handleDelete() {
+    onDeleteEmotion(id);
+    router.push("/");
+  }
 
-  return <EmotionDetails entry={emotionEntry} onBack={() => router.back()} />;
+  return (
+    <EmotionDetails
+      entry={emotionEntry}
+      onDeleteEmotion={handleDelete}
+      onDeleteEmotionBack={() => router.back()}
```

Comment:
We do not necessarily need the onDeleteEmotionBack / router.back-function because we are using our router.push already to navigate back to the HomePage after deletion, so the onDeleteEmotionBack is somewhat redundant I guess. Also in this scenario the EmotionDetails doesn´t receive the prop anyway so I suggest we delete the line 26.